### PR TITLE
Storybook: Box/Margins, Icon, Checkbox

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers this adds more common-adapters to the storybook

### CODE REVIEW

File: shared/common-adapters/box.stories.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// @flow
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+import React from 'react'
+import Box from './box'
+import {storiesOf} from '@storybook/react'
+import {globalMargins, globalStyles, globalColors} from '../styles'
+
+storiesOf('Box', module).add('Box', () => (
```
Comment (chrisnojima) [inline shared/common-adapters/box.stories.js]: show our margins

File: shared/common-adapters/icon.stories.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  const meta = iconMeta[type]
+  const twoRegMatch = type.match(/(\d+)-x-\d+$/)
+  const oneRegMatch = type.match(/(\d+)$/)
+  const size = meta.gridSize || (twoRegMatch && twoRegMatch[1]) || (oneRegMatch && oneRegMatch[1]) || '?'
+
+  if (!sizes[size]) {
+    sizes[size] = []
+  }
+
+  sizes[size].push(type)
```
Comment (chrisnojima) [inline shared/common-adapters/icon.stories.js]: split it out by size

File: shared/constants/platform.desktop.js [hunk 3]
```diff
@@ -78,56 +78,67 @@ const darwinCacheRoot = `${getenv('HOME', '')}/Library/Caches/${envedPathOSX[run
 const darwinSandboxCacheRoot = `${getenv('HOME', '')}/Library/Group Containers/keybase/Library/Caches/${envedPathOSX[runMode]}/`
 const darwinSandboxSocketPath = path.join(darwinSandboxCacheRoot, socketName)
 
-function findSocketDialPath(): Array<string> {
-  const paths = {
-    darwin: darwinSandboxSocketPath,
-    linux: linuxSocketDialPath(),
```
Comment (chrisnojima) [inline shared/constants/platform.desktop.js]: this pattern executes this code even if we're not on this platform, lets not do that

File: shared/stories/index.js [hunk 4]
```diff
@@ -1,13 +1,24 @@
...
 
 // Load css
 import '../desktop/renderer/style.css'
 
+// Load common-adapter stories
 // $FlowIssue
 const req = require.context('../common-adapters', true, /\.stories\.js$/)
 
+// Add material-ui dependency
+addDecorator(story => (
```
Comment (chrisnojima) [inline shared/stories/index.js]: some sub dependencies still rely on material-ui (like icon). So if any component happens to need that it'll fail. as a workaround we wrap all our stories in material. TODO this'll have to be different on mobile

Comment (chrisnojima) [inline shared/stories/index.js]: or we have to remove material-ui (likely better)

File: shared/constants/platform.desktop.js (hunk 3 above)
Comment (MarcoPolo) [inline shared/constants/platform.desktop.js]: gross

File: shared/stories/index.js (hunk 4 above)
Comment (MarcoPolo) [inline shared/stories/index.js]: I thought you did that a while ago, what's left?

**Comment by MarcoPolo** [comment]
This is really great! 😄

File: shared/stories/index.js (hunk 4 above)
Comment (chrisnojima) [inline shared/stories/index.js]: its there in a couple of places (icon, dropdown, menu-list)

**Pull request merged**